src/utils/notifyIssues.js

```diff
@@ -0,0 +1,100 @@
+'use strict'
+
+// const fs = require('fs')
+const pMap = require('p-map')
+
+const { logError, logWarning } = require('../log')
+const { getPrNumbersFromReleaseNotes } = require('./releaseNotes')
+
+async function getLinkedIssueNumbers(octokit, prNumber, repoOwner, repoName) {
+  const data = await octokit.graphql(
+    `
+    query getLinkedIssues($repoOwner: String!, $repoName: String!, $prNumber: Int!) {
+      repository(owner: $repoOwner, name: $repoName) {
+        pullRequest(number: $prNumber) {
+          id
+          closingIssuesReferences(first: 100) {
+            nodes {
+              id
+              number
+            }
+          }
+        }
+      }
+    }
+    `,
+    {
+      repoOwner,
+      repoName,
+      prNumber,
+    }
+  )
+
+  const linkedIssues =
+    data?.repository?.pullRequest?.closingIssuesReferences?.nodes
+
+  if (!linkedIssues) {
+    return []
+  }
+
+  return linkedIssues.map(issue => issue.number)
+}
+
+async function notifyIssues(githubClient, owner, repo, release) {
+  console.log({
```

Comment:
avoid console log, use the action logger utilities

Reply:
the console log is for dev purposes and will be removed from the final version